assets/sass/widgets/_widget-area.scss

```diff
@@ -0,0 +1,42 @@
+/**
+ * Widget Area styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-widget-area--composite.googlesitekit-widget-area-widgets {
```

Comment:
This is still off, there needs to be a space in between. These two classes will never be on the same element, `.googlesitekit-widget-area-widgets` is a child element of the other one (see `WidgetAreaRenderer` component).